backend/packages/Upgrade/src/loaders/typeormLoader.ts

```diff
@@ -67,8 +199,74 @@ export const typeormLoader: MicroframeworkLoader = async (settings: Microframewo
     synchronize: env.db.synchronize,
     logging: env.db.logging as boolean | 'all' | LogLevel[],
     maxQueryExecutionTime: env.db.maxQueryExecutionTime,
-    entities: env.app.dirs.entities,
-    migrations: env.app.dirs.migrations,
+    entities: [
```

Comment:
@RidhamShah Can you make a common array outside and use in both master and export connection, as its redundant.

Reply:
code updated